test/loginTest.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * Created by austin on 30/04/2017.
+ */
+'use strict';
+
+const assert = require('chai').assert;
+const fixtures = require('./fixtures.json');
+const kujiraService = require('./kujira-service.js');
+const user = require('./../../kujira-credentials.json');
```

Comment:
As mentioned in previous PR - create stub file & just relocate to test directory